tests/FSharp.Data.GraphQL.IntegrationTests.Server/JSON.fs

```diff
@@ -0,0 +1,32 @@
+[<CompilationRepresentation (CompilationRepresentationFlags.ModuleSuffix)>]
```

Comment:
This file must be referenced as link from StartWars.
I did that in some branch. We do not want to duplicate